02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+                "summary": "Children under 18 years of age can be certified as a separate household from other adults living in the home if the adults do not exercise parental control or offer financial support, and clients who are officially emancipated usually fall under this category."
+            },
+            "What if a client does not want to provide information about other household members?": {
+                "score_sum": 0.8257677270622369,
+                "derived_questions": [
+                    "Do clients have to list their adult children living with them if they have jobs and buy their own food?"
+                ],
+                "entire_card": "What if a client does not want to provide information about other household members?\nWe always want to try and collect as much information about everyone living in the household, but sometimes clients are not comfortable disclosing information about people living there who are non-applicants.\nIf a client is living with non-mandatory household members, it is helpful to include basic information. When other household members are not included, it can pose delays down the line when caseworkers uncover other household members. When we have an idea of the composition of the household, it allows for faster troubleshooting.\nFOR FOOD STAMP BENEFITS\nIn scenarios where there are people who are\nNOT in the client's SNAP household\n(roommates/boarders/family members who share less than 50% of the grocery budget) ideally, you will want to collect full name, birth date, gender and marital status for everyone not applying.  You will\nat least\nneed to collect names and approximate ages of other people in the household.\nWe can proceed with the application if the client is insistent about not including them. We should still leave a portfolio note explaining that other household members reside with the client, but the client requested they not be included.\nFor example: Client has 3 additional roommates, but requested they not be listed on the application. Client does not share a food budget with the other members and they are not mandatory household members.",
+                "summary": "It is recommended to collect basic information about non-applicant household members, even if the client does not want to include them on the application, in order to avoid delays and facilitate troubleshooting."
```

Comment:
**Relevance to the question**: 🟠 Potentially relevant
- Relevant because it speaks to the fact that the client does not want to include the child on the application, but misleading because the child is a mandatory household member.

**Correctness as a summary of the Guru card:** 🟡 Missing Info 
- The summary is correct but the Guru card provides specific guidance about the minimal fields to collect, as well as guidance about how to proceed if the client insists on not including additional household members

**Helpfulness**: 🟠 Unhelpful
- The summary (taken alone) could mislead the user to believe (incorrectly) that the 20yo son can be considered a non-applicant household member, however it could be partly helpful in the instance that the client refused to include the child in the application.